common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -37,6 +37,10 @@
   private static Method sCleanerCleanMethod;
   private static Method sByteBufferCleanerMethod;
 
+  public static int byteToInt(byte b) {
+    return (int) b & 0xFF;
```

Comment:
Are you trying to cast the byte to an int before the & or casting the result to an int (either way I'm not sure if its necessary)?